src/main/java/com/atlassian/bitbucket/jenkins/internal/client/BitbucketFilePathClientImpl.java

```diff
@@ -49,7 +54,7 @@ public List<SCMFile> getDirectoryContent(BitbucketSCMFile directory) {
     }
 
     @Override
-    public String getFileContent(BitbucketSCMFile file) {
+    public String getFileContent(BitbucketSCMFile file) throws IOException {
         HttpUrl url = getUrl(file);
```

Comment:
I think should use the streaming endpoint and gather the contents up in a `String` at the end, but I think we don't even need it any more, so I lean more towards just removing this along with the `FileNextPageFetcher` and `BitbucketFilePage` objects